docs/collecting-data/collecting-from-own-applications/flutter-tracker/anonymous-tracking/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Anonymous tracking"
+date: "2022-01-31"
+sidebar_position: 6000
+---
+
+Anonymous tracking is a tracker feature that enables anonymising various user and session identifiers to support user privacy in case consent for tracking the identifiers is not given.
```

Comment:
Could you use this instead? https://github.com/snowplow/documentation/blob/main/docs/reusable/anonymous-tracking-mobile/_index.md

See https://github.com/snowplow/documentation/blob/main/docs/collecting-data/collecting-from-own-applications/mobile-trackers/anonymous-tracking/index.md for how to import it.

Reply:
That snippet only talks about the mobile trackers, here I'm including info about web as well

Reply:
Does it make sense to amend the snippet so that it fits all 3 pages where it could be used?

Reply:
I've separated out mobile and web info for Flutter and changed the reusable block